mailet/api/src/main/java/org/apache/mailet/Attribute.java

```diff
@@ -32,8 +32,12 @@
  * @since Mailet API v3.2
  */
 public class Attribute {
-    public static Attribute convertToAttribute(String name, Object value) {
+    public static Attribute of(String name, Object value) {
         AttributeName attributeName = AttributeName.of(name);
+        return of(attributeName, value);
+    }
+
+    public static Attribute of(AttributeName attributeName, Object value) {
         AttributeValue<?> attributeValue = AttributeValue.ofAny(value);
```

Comment:
We should prevent a `AttributeValue<AttributeValue<?>>`.

Reply:
Then let's add a Precondition to enforce safety while keeping the cool API

Reply:
Mmmh, I'm was wondering about this, I think that forcing the developer to have strong type earlier as possible is a better policy.

Reply:
The downside being boilerplate.